File: CS50x_conputer_science/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

//FUCTIONS PROTOTYPES
int userInput(void);
int draw(int height);

int main(void)
{
    int height = userInput();
    draw(height);
}

//FUNCTIONs DECLARATIONS

//GET USER INPUT
int userInput(void)
{
    int height;
    do
    {
        height = get_int("what't the height?\n");
    }
    while (height < 1 || height > 8);
    return height;
}

//DRAW
int draw(int height)
{
    //PRINT SPACES BEFORE
    for (int i = 0; i < height; i++)
    {
        for (int s = height - 1; s > i; s--)
        {
            printf(" ");
        }

        //PRINT FRIST ROW OF "#"
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        //PRINT MIDDLE SPACES
        printf("  ");

        //PRINT LAST ROW OF "#"
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        printf("\n");
    }
    return 1;
}

File: CS50x_computer_science/sentimental-mario-less/mario.py
from cs50 import get_int


# get height
while True:
    try:
        height = get_int("Height: ")
        if (height > 0) and (height <= 8):
            break
    except:
        print("", end="")

spaces = 1
# print newline
for i in range(height):

    # print spaces
    for spaces in range(height-i-1):
        print(" ", end="")

    # prints "#"
    for j in range(i+1):
        print("#", end="")

    print()